lib/inventory/controller.go

```diff
@@ -435,6 +435,43 @@ func (c *Controller) handleHeartbeatMsg(handle *upstreamHandle, hb proto.Invento
 			return trace.Wrap(err)
 		}
 	}
+
+	if hb.CommandLabels != nil || hb.ImportedLabels != nil {
+		if err := c.handleLabelHBs(handle, hb); err != nil {
+			return trace.Wrap(err)
+		}
+	}
+
+	return nil
+}
+
+func (c *Controller) handleLabelHBs(handle *upstreamHandle, hb proto.InventoryHeartbeat) error {
+	tracker := &handle.stateTracker
+	tracker.mu.Lock()
+	defer tracker.mu.Unlock()
+
+	if hb.CommandLabels != nil {
+		for key, id := range tracker.labels.CommandKeys {
+			log.Infof("---> Command label key: %q -> %d", key, id)
```

Comment:
Would debug level be more appropriate for these messages since we're logging on every heartbeat?